### ISSUE TITLE
Otter does not remember position in queue (always start from top)

### ISSUE THREAD

**Issue by gnouts** [apognu/otter#90]
My use case : I mostly use my phone with bluetooth device (car or headphone). When I connect device, Otter start, fantastic. When I disconnect Otter stop, fantastic. When I reconnect to device, Otter start but from the first track in the queue, which I just listen to.
Coming from DSub, it remember the position where I left and start from there.
I don't reproduce "manually", I mean without a bluetooth device.

**How to reproduce**

1. Add multiple tracks to queue, like a playlist.
2. Connect to a bluetooth device and start playing multiple songs
3. Disconnect bluetooth device
4. Reconnect bluetooth device
5. Track playing is the first you add to queue and not the last you listen.

**Expected behavior**

Otter should start playback from where I left when I disconnect bluetooth.

**Environment details**

 - Device: fairphone2, bluetooth radio from my car or Plantronics headset
 - Android version: FairphoneOS, Android 7.1.2
 - App version: 1.0.21
 - If public Funkwhale instance, its URL:

**Comment by DanielNautre** [comment]
Hi,

I have a similar issue though in my case, the queue always starts from the top when opening it. A more natural behavior would be to open it from the currently playing track.

![Screenshot_20201007-112007](https://user-images.githubusercontent.com/2039600/95313111-0b21a800-0890-11eb-92e8-3492142af6cb.jpg)



**Environment details**
Device: Samsung S7
Android version: Android 8.0.0
App version: 1.0.21

**Comment by gnouts** [comment]
@DanielNautre If I understand your report it's not only related to bluetooth connection ?

**Comment by DanielNautre** [comment]
@gnouts Correct, in my case, everytime I open the queue it shows it from the start.

**Comment by AMoonRabbit** [comment]
Do you get any crash logs when this has happened? For me, it's remembered it's position in a queue unless the app has a soft crash.

I will note that the playlist drawer does not re-open around the currently playing track, the view will always been from the top of the list, and you have to scroll down to where the current track is selected. But in terms of playback, it will usually resume where it left off.

**Comment by DanielNautre** [comment]
Seems there is two different issues here. The one from @gnouts where the music starts playing from the start of the queue. The second issue is purely UI where opening the queue displays from the top of the list. I will open a separate issue for this problem to avoid confusion: #95
